app/build.gradle

```diff
@@ -115,21 +115,21 @@ dependencies {
 
     compile 'com.jakewharton.timber:timber:4.1.2'
     compile "com.jakewharton:butterknife:$BUTTERKNIFE_VERSION"
-    apt     "com.jakewharton:butterknife-compiler:$BUTTERKNIFE_VERSION"
+    annotationProcessor "com.jakewharton:butterknife-compiler:$BUTTERKNIFE_VERSION"
 
     // Replace provided dependency below with official AutoValue once this issue is fixed
     // https://github.com/google/auto/issues/268
     provided 'com.jakewharton.auto.value:auto-value-annotations:1.2-update1'
-    apt      "com.google.auto.value:auto-value:$AUTO_VALUE_VERSION"
-    apt      'com.ryanharter.auto.value:auto-value-parcel:0.2.3-rc2'
-    apt      'com.ryanharter.auto.value:auto-value-gson:0.3.2-rc1'
-    apt      'com.squareup:javapoet:1.7.0' // https://github.com/rharter/auto-value-parcel/issues/64
+    annotationProcessor      "com.google.auto.value:auto-value:$AUTO_VALUE_VERSION"
+    annotationProcessor      'com.ryanharter.auto.value:auto-value-parcel:0.2.3-rc2'
+    annotationProcessor      'com.ryanharter.auto.value:auto-value-gson:0.3.2-rc1'
+    annotationProcessor      'com.squareup:javapoet:1.7.0' // https://github.com/rharter/auto-value-parcel/issues/64
```

Comment:
Just to leave it a little tidier, we should remove this extra spacing. Previously there was extra spacing after 'apt' to improve readability by keeping the imports aligned with the 'provided' import above, but now 'annotationProcessor' takes up more space than 'provided' so the extra spacing does not serve any purpose.